task/06-conditions-n-loops-tasks.js

```diff
@@ -81,11 +101,13 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,1,1   =>  false
  *   10,10,10 =>  true
  */
-function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+function isTriangle(a, b, c) {
+    if(a+b>c&&a+c>b&&b+c>a){
+        return true
+    }
+    return false
```

Comment:
code style and improve logic please
we can just `return a + b > c && a + c > b && b + c > a`
Please, see https://javascript.info/logical-operators